server/Forms.py

```diff
@@ -19,4 +20,10 @@ class SignUpForm(FlaskForm):
                         render_kw={"placeholder": "Email"})
     password = PasswordField('', 
                             validators=[InputRequired(), Length(min=6, max=80)],
-                            render_kw={"placeholder": "Password"})
\ No newline at end of file
+                            render_kw={"placeholder": "Password"})
+
+class TrainingForm(FlaskForm):
+    file = FileField('image', validators=[
+        FileRequired(),
+        FileAllowed(['jpg', 'jpeg'], 'Allowed file formats: .jpg, .jpeg!')
+    ])
```

Comment:
This is the file field in the form, you can easily add validation here using the built-in functions.